nautobot/core/checks.py

```diff
@@ -44,6 +51,16 @@ def check_object_permissions_backend(app_configs, **kwargs):
     return []
 
 
+@register(Tags.security)
+def check_sso_install(app_configs, **kwargs):
+    if (
+        sso_auth_enabled(auth_backends=settings.AUTHENTICATION_BACKENDS)
+        and "social_django" not in settings.INSTALLED_APPS
```

Comment:
Forgot to add, we can probably remove this check and I suppose we could perhaps even remove the system check since we're including `social_django` by default?